.travis.yml

```diff
@@ -4,15 +4,26 @@ sudo : false
 
 matrix:
   include:
-    - python: 3.5
+    - python: "3.5"
       env: TOXENV=py35
+  include:
+    - python: "3.6"
+      env: TOXENV=py36
+  include:
+    - python: "3.7"
+      dist: xenial
+      sudo: true
+      env: TOXENV=py37
+  include:
+    - python: "pypy"
+      env: TOXENV=pypy
+  include:
+    - python: "pypy3"
+      env: TOXENV=pypy3
```

Comment:
Is there a reason for the more complicated `matrix:` dance here, versus just including all the supported Python versions in `env:`?

Reply:
I tried it before in the first commit, but there seems a problem with `travis_retry tox` in this combination, see travis log. One alternative is to change the setup more to the style currently used in the zopefoundation projects like https://github.com/zopefoundation/z3c.flashmessage/pull/2.

What do you prefer?